news/vox-remove.rst

```diff
@@ -0,0 +1,13 @@
+**Added:** 
+
+* `vox remove` command can remove multiple environments at once.
```

Comment:
I think that you will need to create a new PR, since this one was already merged. Thanks!